[2025-01-29T16:22:52.848+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T16:22:52.898+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.location_Transform manual__2025-01-29T16:22:08.198012+00:00 [queued]>
[2025-01-29T16:22:52.918+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.location_Transform manual__2025-01-29T16:22:08.198012+00:00 [queued]>
[2025-01-29T16:22:52.919+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-01-29T16:22:52.944+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): location_Transform> on 2025-01-29 16:22:08.198012+00:00
[2025-01-29T16:22:52.992+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2074) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T16:22:53.006+0000] {standard_task_runner.py:72} INFO - Started process 2087 to run task
[2025-01-29T16:22:53.010+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'location_Transform', 'manual__2025-01-29T16:22:08.198012+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpwtdzwbz6']
[2025-01-29T16:22:53.028+0000] {standard_task_runner.py:105} INFO - Job 94: Subtask location_Transform
[2025-01-29T16:22:53.235+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.location_Transform manual__2025-01-29T16:22:08.198012+00:00 [running]> on host ae52aba2dca2
[2025-01-29T16:22:53.532+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='itsMe!!' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='location_Transform' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T16:22:08.198012+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T16:22:08.198012+00:00'
[2025-01-29T16:22:53.535+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T16:22:54.713+0000] {logging_mixin.py:190} INFO - <class 'pandas.core.frame.DataFrame'>
Index: 399 entries, 0 to 2563
Data columns (total 5 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   Postal Code  399 non-null    int64 
 1   Country      399 non-null    object
 2   Region       399 non-null    object
 3   State        399 non-null    object
 4   City         399 non-null    object
dtypes: int64(1), object(4)
memory usage: 18.7+ KB
[2025-01-29T16:22:54.714+0000] {logging_mixin.py:190} INFO - None
[2025-01-29T16:22:56.020+0000] {logging_mixin.py:190} INFO - !_upload_to_S3_complete_!
[2025-01-29T16:22:56.032+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T16:22:56.101+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T16:22:56.104+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=location_Transform, run_id=manual__2025-01-29T16:22:08.198012+00:00, execution_date=20250129T162208, start_date=20250129T162252, end_date=20250129T162256
[2025-01-29T16:22:56.211+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T16:22:56.394+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-29T16:22:56.401+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
